pyspecProcScripts/third_level/rough_table_of_integrals.py

```diff
@@ -54,6 +54,7 @@ def rough_table_of_integrals(
     fig.set_figheight(6)
     # {{{ set up subplots
     fl.next("Raw Data with averaged scans", fig=fig)
+    fl.skip_units_check()
```

Comment:
you need to update pyspecdata to get this -- we'll want to do this whenever we load multiple subplots.